### DISCUSSION TITLE
Set property name/id for each text annotation

### DISCUSSION THREAD

**Discussion by OtteryFox** [stephanrauh/ngx-extended-pdf-viewer#2535] [Q&A]
with export annotations I get all the details

   _exportAnnotations(): void { 
    this.rawAnnotations = this.pdfViewerService.getSerializedAnnotations(); 
    console.log(this.rawAnnotations);
  }_

but i want to set property name for that annotation and add extra details like 'max data', 'x', 'y', 'height', 'width' and save.

**Comment by stephanrauh** [comment]
I'm afraid that's too difficult to implement. The annotations are part of the base library. I'm using a fork of that library, but if I modify the wrong part, I end up in merge hell - and I think this is one of the dangerous parts.
